src/services/fs/fs.service.ts

```diff
@@ -1,5 +1,9 @@
 import { Injectable } from "@nestjs/common";
 import fs from "fs";
+import rimraf from "rimraf";
+import { promisify } from "util";
+
+const rm = promisify(rimraf);
```

Comment:
>rimraf [](start = 21, length = 6)

this one seems to be done as a dev dependency

Reply:
also you dont even need to define this class, you can use fs-xtra. [https://www.npmjs.com/package/fs-extra](https://www.npmjs.com/package/fs-extra)

---
In reply to: [300143207](https://github.com/Azure/git-rest-api/pull/51#discussion_r300143207) [](ancestors = 300143207)